libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
good idea.  But the figuredbass.cpp code does not expert everything needed.  I decided to write most of the code from
inside figuredbass.cpp to avoid this issue.  Looks like there is still a bit more juggling to do.

Jim



> On 22 May 2015, at 21:32, Marc Sabatella <notifications@github.com> wrote:
> 
> In libmscore/figuredbass.cpp <https://github.com/musescore/MuseScore/pull/2015#discussion_r30924457>:
> 
> >  #include "rest.h"
> >  #include "score.h"
> >  #include "sym.h"
> >  #include "xml.h"
> > +#include "utils.h"
> > +#include "mscore/scoreview.h"
> You can certainly use the functions, just not from within libmscore. One approach would be to initiate the operation from scoreview, but call functions in score as need. A number of othet commands work that way.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/2015/files#r30924457>.
>